packages/jazz-tools/src/tools/subscribe/SubscriptionScope.ts

```diff
@@ -759,3 +756,13 @@ export class SubscriptionScope<D extends CoValue> {
     this.childNodes.forEach((child) => child.destroy());
   }
 }
+
+function hasAccessToCoValue(rawCoValue: RawCoValue): boolean {
+  const ruleset = rawCoValue.core.verified.header.ruleset;
+
+  // Groups and accounts are accessible by everyone, for the other coValues we use the role to check access
+  return (
+    ruleset.type !== "ownedByGroup" ||
+    myRoleForRawValue(rawCoValue) !== undefined
```

Comment:
Question here, do we also want to avoid running migrations if the user only has read access?

See: https://discord.com/channels/1139617727565271160/1139621689882321009/1445690193213653116